src/NLog.Web.AspNetCore/AspNetExtensions.cs

```diff
@@ -112,8 +112,11 @@ public static LogFactory ConfigureNLog(this ILoggingBuilder builder, LoggingConf
         /// <param name="configFileName">Path to NLog configuration file, e.g. nlog.config. </param>
         public static ILoggingBuilder AddNLog(this ILoggingBuilder builder, string configFileName)
         {
-            ConfigureServicesNLog(null, builder.Services, serviceProvider => serviceProvider.GetService<IConfiguration>());
-            LogManager.LoadConfiguration(configFileName);
+            ConfigureServicesNLog(null, builder.Services, (s, c) =>
```

Comment:
Please use longer names then `s` and `c`. This isn't C ;)

Reply:
Found a better way that got rid of short names.